ibm/service/vpc/resource_ibm_is_bare_metal_server.go

```diff
@@ -696,7 +696,8 @@ func resourceIBMISBareMetalServerCreate(context context.Context, d *schema.Resou
 	if err != nil {
```

Comment:
Does changes in this file belong to this feature

Reply:
no, sdk changes